packages/runtime/src/Runtime.ts

```diff
@@ -191,6 +191,14 @@ export abstract class Runtime<TConfig extends RuntimeConfig = RuntimeConfig> {
         this._anonymousServices = Container.get<AnonymousServices>(AnonymousServices);
     }
 
+    public async validateUsedBackboneVersion(): Promise<Result<void>> {
+        if (!this._accountController) {
```

Comment:
I don't see a reason why this is only possible with an active account.

Reply:
will you change this? YOu thumbed up.

Reply:
yeah, I will